projects/storefrontlib/shared/components/card/card.component.ts

```diff
@@ -44,6 +44,9 @@ export class CardComponent implements OnInit {
   @Input()
   border = false;
 
+  @Input()
```

Comment:
We usually don't want to add more properties as inputs if we don't need to. We already have an input of `content: Card`. Could we add `role`to the Card interface and pass it in with the `content`input?

Reply:
Good catch, I will try